frontend/pages/components/QRscanner.tsx

```diff
@@ -43,6 +45,11 @@ const QRscanner = ({ qrView, onSuccess, setQRview }: QRscannerProps) => {
     }
   }, [qrView]);
 
+  const handleBackButton = () => {
```

Comment:
please. try to use function instead of arrow functions. 
arrow functions block you on the flow of the code. so you cant call them if is not on top of the current line
with normal functions it can be called. no matter the position on code.